containers/compose-base.yaml

```diff
@@ -126,5 +127,30 @@ services:
       retries: 5
       start_period: 40s  # requires docker-compose 3.4
 
+  minio:
+    image: quay.io/minio/minio
+    restart: unless-stopped
+    env_file:
+      - ../.env
+    command: [
+      "server",
+      "/data",
+      "--console-address",
+      ":9090"
+    ]
+    ports:
+      - ${MINIO_PORT_1}:9000
+      - ${MINIO_PORT_2}:9090
+    volumes:
+      - minio:/data
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
```

Comment:
We should open an issue for this health check endpoint to be added to the status check airflow DAG

Reply:
done